### PR TITLE
Reorder extensions

### PR DESCRIPTION
**Pull request by Amerlander**
@joernalraun changed the order of the extensions.
No extensions added.

@riknoll can you approve?

### CODE REVIEW

**Review by riknoll — APPROVED** [review]

File: targetconfig.json [hunk 1]
```diff
@@ -145,19 +156,8 @@
...
-            "calliope-edu/makecode-extension-scd41",
-            "calliope-edu/pxt-HM3301_Dust_Sensor",
-            "calliope-edu/pxt-TCS34725FN",
-            "calliope-edu/pxt-display",
-            "calliope-edu/pxt-grove",
-            "calliope-edu/pxt-rgblcd",
-            "calliope-edu/pxt-sunlightsensor-si1145",
-            "calliope-mini/pxt-SCD30",
             "dl1ekm/pxt-calliope-ADS1x15",
             "dl1ekm/pxt-calliope-PCF85063-RTC",
```
Comment (riknoll) [inline targetconfig.json:160]:
```suggestion
            "dl1ekm/pxt-calliope-PCF85063-RTC"
```

**Comment by riknoll** [comment]
@Amerlander looks like this broke the JSON! I'll commit a fix and merge

**Comment by riknoll** [comment]
ah, actually I think you need to since I don't have access to that repo

**Comment by Amerlander** [comment]
I`m just calling with Jörn about this atm, becouse I also wasnt able to change it :)

**Comment by Amerlander** [comment]
Is now part of PR https://github.com/microsoft/pxt-calliope/pull/198

**Pull request closed without merging**